lib/web_ui/lib/src/engine/compositor/image.dart

```diff
@@ -48,6 +48,9 @@ class SkAnimatedImage implements ui.Image {
   @override
   int get height => _skAnimatedImage.callMethod('height');
 
+  @override
+  int get approximateBytesUsed => 0;
```

Comment:
Here and elsewhere: we could probably fill this in with `width * height * 4`, although not sure how important that is. That also doens't account for mipmapping or images that do not use 4 bytes per pixel.

@yjbanov @ferhatb for thoughts